### DISCUSSION TITLE
When new release in marketplace?

### DISCUSSION THREAD

**Discussion by ChmilevFA** [KartanHQ/intellij-hardhat#68] [Q&A]
I see that some development is happening but plugin is pretty outdated in JetBrains marketplace + not working with new versions of IDE
https://plugins.jetbrains.com/plugin/18551-hardhat
![image](https://user-images.githubusercontent.com/17650184/170711025-f8669675-4843-4694-8293-1b92ffa55db2.png)

Any plans to release new version soon via the marketplace?

**Comment by ChmilevFA** [comment]
@nekofar maybe?

**Reply by nekofar** [reply]
Sorry, I didn't notice your message; I'll be looking into it. Would you please open a [bug report](https://github.com/KartanHQ/intellij-hardhat/issues/new?assignees=&labels=&template=bug_report.md&title=) issue and let me know which version of IntelliJ you are using?
